presto-main/src/main/java/com/facebook/presto/server/security/LdapServerConfig.java

```diff
@@ -20,7 +20,16 @@
     private String ldapUrl;
     private String baseDistinguishedName;
     private boolean authenticationEnabled;
+    private String serverType = ServerType.AD.toString();
```

Comment:
What's the reason for initializing the `serverType` variable? Is it because `authentication.ldap.server-type` configuration  property is not mandatory to be set?